src/tasks/AutoHeistTask.py

```
@@ -58,7 +59,7 @@ def _inst_gap():
         _inst_line("├─ 移动镜头修正：禁用", "#FE821D", bold=True, indent=2),
         _inst_line("└─ 按下锁定镜头回正：启用", "#FE821D", bold=True, indent=2),
         _inst_line("⚠️ 必备条件：至少有一个复活道具", "#FF5555", bold=True),
-        _inst_line("🥷 避战方式：翳【长按 Shift】/ 浔【长按攻击】", "#FF5555", bold=True),
+        _inst_line("🥷 避战方式：翳【长按 Shift】/ 浔【长按攻击】/残虹【点按G】", "#FF5555", bold=True),
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)